lib/bundler/plugin.rb

```diff
@@ -0,0 +1,5 @@
+module Bundler
+  module Plugin
+    autoload :V1,        "bundler/plugin/v1"
```

Comment:
I don't feel like this sort of versioning is necessary -- plugins in their gemspecs could just declare bundler version requirements?